src/components/view/DetailsTab.vue

```diff
@@ -33,6 +33,9 @@
             <router-link :to="{ path: '/volume/' + volume.uuid }">{{ volume.type }} - {{ volume.path }}</router-link> ({{ parseFloat(volume.size / (1024.0 * 1024.0 * 1024.0)).toFixed(1) }} GB)
           </div>
         </div>
+        <div v-else-if="$route.path.startsWith('/project/') && item === 'account' && fetchProjectAdmins()">
```

Comment:
@Pearl1594 is this necessary now, if there's a new tab that has the details?

Reply:
This is necessary in case there are multiple project admins. So as to have the detail view also provide a comma-separated list as done in the list view